src/sentry/static/sentry/app/components/errorBoundary.tsx

```diff
@@ -70,13 +77,14 @@ class ErrorBoundary extends React.Component<Props, State> {
   }
 
   componentWillUnmount() {
+    this._isMounted = false;
     if (this.unlistenBrowserHistory) {
       this.unlistenBrowserHistory();
     }
   }
 
-  // XXX: browserHistory.listen does not have a correct return type.
-  unlistenBrowserHistory: any;
+  unlistenBrowserHistory: ReturnType<typeof browserHistory.listen> | undefined;
+  _isMounted: boolean | undefined;
```

Comment:
Why not just type this as a Boolean and instantiate as false?

Reply:
thanks, simplified the types here a bit